src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -191,7 +191,13 @@ public function convert(FileVersion $fileVersion, $formatKey, $imageFormat)
 
         // Set Interlacing to plane for smaller image size.
         if (1 == count($image->layers())) {
-            $image->interlace(ImageInterface::INTERLACE_PLANE);
+            try {
+                $image->interlace(ImageInterface::INTERLACE_PLANE);
+            } catch (\InvalidArgumentException $exception) {
+                // ignore unsupported Interface type
+            } catch (\RuntimeException $exception) {
```

Comment:
Is there no better exception? Same question for the above `catch`

Reply:
@wachterjohannes It should be a `Imagine\Exception\RuntimeException`

Reply:
@alexander-schranz thats not correct - the Vips Image throws a `\RuntimeException` because the function it is not implemented there.

https://github.com/rokka-io/imagine-vips/blob/0.9.0/lib/Imagine/Vips/Image.php#L514

what you have changed yesterday there 🙈 i will also catch the new exception here.

Reply:
Btw, I am even not sure if they did that on purpose, because sometime they throw the `Imagine\Exception\RuntimeException` as @alexander-schranz mentioned... I think that might be even a bug in their adapter.

Reply:
For clearification in version < 1.0 imagine. Not Implemented or supported function did throw a `Imagine\Exception\RuntimeException`. In >=1.0 they did add a new exception the `Imagine\Exception\NotSupportedException` which extends from `Imagine\Exception\RuntimeException`. Sadly the vips adapter didn't implement that correctly so I created a PR not sure if they gonna release it soon.

Reply:
Let's wait for their answer in rokka-io/imagine-vips#7